requirements/requirements_doc.txt

```diff
@@ -11,5 +11,5 @@ sphinx-gallery==0.11.1
 sphinx-notfound-page==0.8.3
 sphinxcontrib-websupport==1.2.4
 sphinxemoji==0.2.0
-pyvista>=0.39.0
+pyvista[trame]>=0.39.0
```

Comment:
@prmukherj This is to fix the error related to trame

Reply:
Great, Thank you @mkundu1